README.md

```diff
@@ -27,16 +41,25 @@ pub run build_runner build --output <output directory>
 Run a local development server with a file watcher and incremental rebuilds:
 
 ```
-pub run build_runner serve
+pub run build_runner serve web
 ```
 
-Both of the build and serve commands can use `--config release` to build with
-dart2js instead of the the default dartdevc.
+__Known Issues:__
 
-[build_runner]: https://pub.dartlang.org/packages/build_runner
-[build]: https://pub.dartlang.org/packages/build
+When building and viewing the gallery these issues are expected at this time:
 
-### Test
+ * Build Warnings:
 
-Run `dart tool/sanity_check.dart` to execute a simple Selenium test of the
-gallery.
+   `Generator does not support multiple @GallerySectionConfig annotations in a
+   single lib directory.`
+
+ * Runtime Warning:
+
+   `SEVERE: OverlayService must be a singleton: Check that there is no nested
+   overlayBindings or popupBindings`
+
+ * Loading of deferred libraries are not yet supported when compiling with
+   _dart2js_ and _build_runner_.
```

Comment:
> when compiling with _dart2js_ and _build_runner_.

This gives the impression that it is possible to build w/o using build_runner, which isn't the case, right (if so, maybe drop "and _build_runner_")?

Reply:
It is possible to build with build_runner and DDC but that just ignores deferred loading so I'm not sure.

Reply:
Right, so the bullet could read more simply:

> Loading of deferred libraries are not yet supported when compiling with _dart2js_.

Reply:
Ahh I see what you mean, except you can run the build with build_runner without a web compiler, force the angular builders to run, and output to a directory. Then if you manually run dart2js on the output it will work. This is how I am going to deploy the hosted version until it is working via build_runner. I don't want to make it sound like it's an issue with dart2js. 

I don't think it is worth noting all of that in the README so I'll clean it up the message a little.